src/style.css

```diff
@@ -0,0 +1,116 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+.card {
+  width: min-content;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  font-family: "Roboto", sans-serif;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.titleWrapper {
+  margin-top: 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__productCode {
+  margin-top: 4px;
+  margin-bottom: 10px;
+}
+
+.productCode {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
+}
+  
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background: url(./images/star.svg) no-repeat;
+}
+
+.stars--5 .stars_star,
+.stars--1 .stars_star:first-child,
+.stars--2 .stars_star:nth-child(-n+2),
+.stars--3 .stars_star:nth-child(-n+3),
+.stars--4 .stars_star:nth-child(-n+4) {
+  background: url(./images/star-active.svg) no-repeat;
+}
+
+.card__reviewsWithStars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.numberOfReviews {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.priceWord {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.priceNumber {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.buyButtonWrapper {
+  margin-bottom: 16px;
+}
+
+.card__buyButton {
+  display: flex;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  color: white;
+  text-transform: uppercase;
+  align-items: center;
+  justify-content: space-around;
+}

```

Comment:
add transition for changing color and background-color
https://developer.mozilla.org/en-US/docs/Web/CSS/transition